test/Sentry.Tests/ScopeTests.cs

```diff
@@ -98,7 +98,8 @@ public void Clone_NewScope_IncludesPropagationContext()
 
         var clone = sut.Clone();
 
-        Assert.Same(propagationContext, clone.PropagationContext);
+        Assert.Equal(propagationContext.TraceId, clone.PropagationContext.TraceId);
+        Assert.Equal(propagationContext.SpanId, clone.PropagationContext.SpanId);
```

Comment:
Maybe also assert that the references are not eqal (i.e. this is a clone... not the same object)?